lxpanel-plugin/kano_updater.c

```diff
@@ -8,7 +8,8 @@
 
 #include <gtk/gtk.h>
 #include <gdk/gdk.h>
-#include <glib/gi18n.h>
+#define GETTEXT_PACKAGE "kano-updater"
+#include <glib/gi18n-lib.h>
```

Comment:
You just compile it. The program will read the language file it needs from /usr/locale at runtime. If it doesn't find any for its locale, it doesn't complain and just shows the original strings. In other words, you can distribute this version without language files and it will just work as it does now.